geode-core/src/test/java/org/apache/geode/cache/query/internal/QueryConfigurationServiceImplTest.java

```diff
@@ -43,49 +54,70 @@
 
 @RunWith(JUnitParamsRunner.class)
 public class QueryConfigurationServiceImplTest {
+  private static final Set<String> EMPTY_SET = Collections.emptySet();
+  private CqService mockCqService;
   private InternalCache mockCache;
   private SecurityService mockSecurity;
   private QueryConfigurationServiceImpl configService;
 
+  @Rule
+  public RestoreSystemProperties restoreSystemProperties = new RestoreSystemProperties();
+
   @Before
   public void setUp() {
-    mockCache = mock(InternalCache.class);
+    mockCqService = mock(CqService.class);
+    when(mockCqService.getAllCqs()).thenReturn(Collections.emptyList());
+
     mockSecurity = mock(SecurityService.class);
-    when(mockCache.getSecurityService()).thenReturn(mockSecurity);
     configService = spy(new QueryConfigurationServiceImpl());
+
+    mockCache = mock(InternalCache.class);
+    when(mockCache.getCqService()).thenReturn(mockCqService);
+    when(mockCache.getSecurityService()).thenReturn(mockSecurity);
+  }
+
+  @SuppressWarnings("unused")
+  private Object[] getMethodAuthorizerClasses() {
+    return new Object[] {
+        RegExMethodAuthorizer.class,
+        RestrictedMethodAuthorizer.class,
+        UnrestrictedMethodAuthorizer.class,
+        JavaBeanAccessorMethodAuthorizer.class,
+    };
+  }
+
+  @SuppressWarnings("deprecation")
+  private void setAllowUntrustedMethodInvocationSystemProperty() {
+    System.setProperty(ALLOW_UNTRUSTED_METHOD_INVOCATION_SYSTEM_PROPERTY, "true");
   }
 
   @Test
   public void initThrowsExceptionWhenCacheIsNull() {
     assertThatThrownBy(() -> configService.init(null))
-        .isInstanceOf(IllegalArgumentException.class);
+        .isInstanceOf(IllegalArgumentException.class)
+        .hasMessage("Cache must not be null");
```

Comment:
If this verifcation of the message is being done, then the string used to generate the message in `QueryConfigurationServiceImpl` should be extracted to a constant and referenced here.

Reply:
Done!.